src/modules/aws_sqs/sql/after_install.sql

```diff
@@ -0,0 +1,21 @@
+CREATE FUNCTION
+  before_queue_insert_update () RETURNS TRIGGER LANGUAGE PLPGSQL AS $$
+BEGIN
+    -- can't create a non-fifo queue ending with .fifo and vice-versa
+    IF (NEW.name NOT LIKE '%.fifo') AND (NEW.fifo_queue) THEN
+        RAISE EXCEPTION 'The name of a fifo queue should end with .fifo';
+    END IF;
+    IF (NEW.name LIKE '%.fifo') AND (NOT NEW.fifo_queue) THEN
+        RAISE EXCEPTION 'Non-fifo queue name should not end with .fifo';
+    END IF;
```

Comment:
I *understand* why this is here, but wtf AWS. I am tempted to say "just auto add the `.fifo` prefix when needed and only ban having it explicitly in here" but that could cause confusion for other modules needing to reference this name so...